Apps/Database/src/Models/Form.cs

```diff
@@ -26,15 +26,24 @@ public class Form : AuditableEntity
         [Column("FormId")]
         public Guid Id { get; set; }
 
-        [Required]
-        public DrugProduct DrugProduct { get; set; }
-
         public int PharmaceuticalFormCode { get; set; }
 
         [MaxLength(40)]
         public string PharmaceuticalForm { get; set; }
 
         [MaxLength(80)]
         public string PharmaceuticalFormFrench { get; set; }
+
+        /// <summary>
+        /// Gets or sets the object Drug Product reference.
+        /// </summary>
+        [Required]
+        public virtual DrugProduct DrugProduct { get; set; }
```

Comment:
if the drug product has a reference to this table, is it needed here?

Reply:
This is how foreign key relationships are enforced.